.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -18,6 +18,7 @@ Before we can merge this PR, please make sure that all the following items have
 checked off. If any of the checklist items are not applicable, please leave them but
 write a little note why.
 
+- [ ] If the pr is for none-code change, please submit this it with the message containing a key word like`[skip ci]`, `[skip actions]`, or `[no ci]`.
```

Comment:
```suggestion
```

I don't think this will work for PRs, from the docs
> You won't be able to merge the pull request if your repository is configured to require specific checks to pass first. To allow the pull request to be merged you can push a new commit to the pull request without the skip instruction in the commit message.

I would recommend to add the line to the comment in line 10 like so
```
<!-- Add a description of the changes that this PR introduces and the files that are the most critical to review.
Please commit refactors and minor intermediate (doc) changes on your PR with the `[no ci]` keyword to skip meaningless runs of the full CI pipeline
-->
```

And I'd say we recommend using only a single keyword style, lets say `no ci` so we can easily grep the commit history for these